pkg/xray/resource_xray_security_policy.go

```diff
@@ -51,6 +52,19 @@ func resourceXraySecurityPolicyV2() *schema.Resource {
 				},
 			},
 		},
+		"vulnerability_ids": {
+			Type:        schema.TypeSet,
+			Optional:    true,
+			Set:         schema.HashString,
```

Comment:
For simple type list strings, we don't need to set this attribute. This is already defaulted to `schema.HashString`